website/docs/error-reference.md

```diff
@@ -120,6 +120,18 @@ class Main
 end
 ```
 
+## 4012
+
+This error occurs when an object is defined as class and a method.
+An example is ActiveStorage::Attachment where it is defined
+as "class ActiveStorage::Attachment < ActiveRecord::Base" in
+
+ * sorbet/rails-rbi/models/active_storage/attachment.rbi
+ 
+and module ("def attachment; end") in
+
+ * sorbet/rbi/sorbet-typed/lib/activestorage/all/activestorage.rbi
+ 
```

Comment:
Can you replace your example with this:

https://sorbet.run/#%23%20typed%3A%20true%0Aclass%20A%3B%20end%0Amodule%20A%3B%20end%0A%0Amodule%20B%3B%20end%0Aclass%20B%3B%20end

and include both a code block and a link to the sorbet.run?

Reply:
Ideally, the error-reference should give some sort of troubleshooting  advice for how to fix the error.

Reply:
Actually, there are more things that could trigger this error:

https://sorbet.run/#%23%20typed%3A%20true%0Aclass%20A%3B%20end%0Amodule%20A%3B%20end%0A%0Amodule%20B%3B%20end%0Aclass%20B%3B%20end%0A%0Aclass%20C%3B%20end%0AC%20%3D%201%0A%0AD%20%3D%201%0Aclass%20D%3B%20end

Reply:
Just added your suggestions. Thanks